src/OrbitGl/App.cpp

```diff
@@ -1417,7 +1413,7 @@ void OrbitApp::StartCapture() {
     if (user_defined_capture_data.ContainsFrameTrack(function)) {
       frame_track_function_ids.insert(function_id);
     }
-    selected_functions_map[function_id++] = function;
+    selected_functions_map.insert_or_assign(function_id++, function);
```

Comment:
@beckerhe this is another change that is not just renaming. We can't use [] as FunctionInfo is not default constructible here.